docs/output.md

```diff
@@ -55,6 +55,15 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
 <details markdown="1">
 <summary>Output files</summary>
 
+- `transformation/`
+  - A CSV-formatted file for reporting to IRIDA Next the populated column for each sample: `transformation.csv`
+  - A user-intended CSV-formatted file for reference: `result.csv`
+
+### Categorize
+
+<details markdown="1">
+<summary>Output files</summary>
+
 - `transformation/`
   - A CSV-formatted file for reporting to IRIDA Next the populated column for each sample: `transformation.csv`
```

Comment:
I think likewise here (populated column I think should be specific to the populate transformation?).